test/e2e/data/infrastructure-docker/v1beta1/clusterclass-quick-start.yaml

```
@@ -39,20 +39,27 @@ spec:
       openAPIV3Schema:
         type: string
         default: "k8s.gcr.io"
+        example: "k8s.gcr.io"
+        description: "imageRepository is the container registry to pull images from. If empty, `k8s.gcr.io` will be used by default."
   - name: etcdImageTag
     required: true
     schema:
       openAPIV3Schema:
         type: string
         default: ""
+        example: "3.5.1"
```

Let's use `3.5.1-0`, `3.5.1` does not exist. (full image for tests: `k8s.gcr.io/etcd:3.5.1-0`)
```suggestion
        example: "3.5.1-0"
```

---

Same in the other ClusterClass